lib/http3/common.c

```diff
@@ -587,22 +587,24 @@ static void process_packets(h2o_quic_ctx_t *ctx, quicly_address_t *destaddr, qui
         khiter_t iter = kh_get_h2o_quic_acceptmap(ctx->conns_accepting, accept_hashkey);
         if (iter == kh_end(ctx->conns_accepting)) {
             /* a new connection for this thread (at least on this process); accept or delegate to newer process */
-            if (ctx->acceptor == NULL) {
-                /* This is the offending thread but it is not accepting, which means that the process (or the thread) is
-                 * gracefully shutting down.  Let the application process forward the packet to the next generation. */
+            if (ctx->acceptor != NULL) {
+                if (!quicly_is_supported_version(packets[0].version)) {
+                    send_version_negotiation(ctx, srcaddr, packets[0].cid.src, destaddr, packets[0].cid.dest.encrypted,
+                                             quicly_supported_versions);
```

Comment:
Am I reading this correctly that doing this would send all supported versions? If so I don't think that's what we want. We want to send a VN with only the greasing version.

Reply:
I think this branch is the case where we are still accepting new connections (`ctx->acceptor != NULL`), so telling all supported version is fine.